bsip-0048.md

```diff
@@ -10,45 +10,75 @@
 
 # Abstract
 
-The possibility of creating assets on the BitShares Blockchain is a core feature
-of our platform. In the case of an ico, the ico-holders can create an asset with
-a fixed maximum supply. However this maximum supply can be changed by the
-asset creator (issuer) anytime. This exposes participators ico to the risk,
-of loosing value of their bought asset.
+The possibility of creating assets on the BitShares Blockchain is a core feature of our platform. However some things are too malleable when they should not be. Other things are not flexible when they should be.
 
-With this BSIP a flag will be introduced which forbids the modification of the maximum
-supply of an issuer created asset.
+Asset creators can create an asset with a maximum supply. However this maximum supply can be changed by the asset creator (issuer) anytime. This exposes token holders to the risk of their purchased asset losing value.
+
+With this BSIP a flag will be introduced which forbids the modification of the maximum supply of an issuer created asset.
+
+In addition, another new flag will be introduced that prevents any more tokens from being issued in the future.
+
+And finally, the ability to change the precision of an asset will now be available as long as current supply is zero.
 
 # Motivation
 
-The implementation of a flag, to disable the modification of the maximum supply for 
-issuer created asset, has effects for both sides. On the one side the security for the
-participators or non-issuer users increases, because the value of their assets (tokens)
-can't be manipulated by exposing the market to more created tokens. And on the other side
-it is good marketing for the ico-hoders, because they can advertise with their fixed
-maximum supply.
+A flag to disable the modification of the maximum supply for issuer created asset will be implemented. This has benefits for both the participants and the asset creator. Asset holders are assured the value of their assets (tokens) will not be diluted by issuers creating more tokens. Asset creators are able to market the fact that the maximum supply is fixed.
+
+A flag to disable issuing allows asset holders to be assured that the current supply will remain fixed, even if there are unissued tokens.
```

Comment:
Supply can still decrease, so it won't remain fixed.

Reply:
Fixed.